docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -2033,6 +2033,14 @@ Custom install/learn more URL for enterprise plugins. Defaults to https://grafan
 
 Enter a comma-separated list of plugin identifiers to hide in the plugin catalog.
 
+### public_key_retrieval_disabled
+
+Disable download of public key for verifying plugin signature on startup. Default is `false`. If disabled, it will use the hardcoded public key.
```

Comment:
```suggestion
Disable download of a public key for verifying plugin signature on startup. The default is `false`. If disabled, it will use the hardcoded public key.
```

Reply:
nit: this would be `the public key` since there is only one